tool/github.sh

```diff
@@ -40,8 +40,10 @@ pub get
 if [ "DART_BOT" = "$BOT" ] ; then
 
   # Analyze the Dart code in the repo.
+  chmod 000 flutter-idea # symlinks seem to be causing problems
```

Comment:
cc @scheglov (I'm not at all sure that this is related to the recent analysis context changes, but FYI in any case)

Reply:
@stevemessick - do you have a link to a CI failure (or details about what the problem is)?